features/text/0001-multiple_service_endpoints.md

```diff
@@ -0,0 +1,323 @@
+  </HOSTDN>
+  <BETA>N</BETA>
+  <SERVICE_TYPE>b2handle.handle.api</SERVICE_TYPE>
+  <HOST_IP>62.217.127.244</HOST_IP>
+  <HOST_IPV6>2001:648:2ffc:121:dcdb:eeff:fed6:ba9e/64</HOST_IPV6>
+  <CORE/>
+  <IN_PRODUCTION>Y</IN_PRODUCTION>
+  <NODE_MONITORED>Y</NODE_MONITORED>
+  <SITENAME>GRNET</SITENAME>
+  <COUNTRY_NAME>Greece</COUNTRY_NAME>
+  <COUNTRY_CODE>GR</COUNTRY_CODE>
+  <ROC_NAME>EUDAT_REGISTRY</ROC_NAME>
+  <URL>https://epic.grnet.gr/api/v2/handles/11239</URL>
+  <ENDPOINTS/>
+  <EXTENSIONS>
+    <EXTENSION>
+      <LOCAL_ID>1</LOCAL_ID>
+  <KEY>username</KEY>
+  <VALUE>user</VALUE>
+  </EXTENSION>
+    <EXTENSION>
+      <LOCAL_ID>2</LOCAL_ID>
+  <KEY>epic_version</KEY>
+  <VALUE>2.0</VALUE>
+  </EXTENSION>
+  <EXTENSION>
+  <LOCAL_ID>3</LOCAL_ID>
+  <KEY>handle_version</KEY>
+  <VALUE>7.2.3</VALUE>
+  </EXTENSION>
+  </EXTENSIONS>
+</SERVICE_ENDPOINT>
+
+<SERVICE_ENDPOINT PRIMARY_KEY="34JK">
+  <PRIMARY_KEY>34JK</PRIMARY_KEY>
+  <HOSTNAME>epic.grnet.gr</HOSTNAME>
+  <GOCDB_PORTAL_URL>
+    https://creg.eudat.eu/view_portal/index.php?Page_Type=Service&id=384
+  </GOCDB_PORTAL_URL>
+  <BETA>N</BETA>
+  <SERVICE_TYPE>b2handle.handle.api</SERVICE_TYPE>
+  <HOST_IP>62.217.127.244</HOST_IP>
+  <HOST_IPV6>2001:648:2ffc:121:dcdb:eeff:fed6:ba9e/64</HOST_IPV6>
+  <CORE/>
+  <IN_PRODUCTION>Y</IN_PRODUCTION>
+  <NODE_MONITORED>Y</NODE_MONITORED>
+  <SITENAME>GRNET</SITENAME>
+  <COUNTRY_NAME>Greece</COUNTRY_NAME>
+  <COUNTRY_CODE>GR</COUNTRY_CODE>
+  <ROC_NAME>EUDAT_REGISTRY</ROC_NAME>
+  <URL>https://epic.grnet.gr/api/v2/handles/11500</URL>
+  <ENDPOINTS/>
+  <EXTENSIONS/>
+</SERVICE_ENDPOINT>
+
+<SERVICE_ENDPOINT PRIMARY_KEY="45V3">
+  <PRIMARY_KEY>45V3</PRIMARY_KEY>
+  <HOSTNAME>epic.grnet.gr</HOSTNAME>
+  <GOCDB_PORTAL_URL>
+    https://creg.eudat.eu/view_portal/index.php?Page_Type=Service&id=384
+  </GOCDB_PORTAL_URL>
+  <BETA>N</BETA>
+  <SERVICE_TYPE>b2handle.handle.resolver</SERVICE_TYPE>
+  <HOST_IP>62.217.127.244</HOST_IP>
+  <CORE/>
+  <IN_PRODUCTION>Y</IN_PRODUCTION>
+  <NODE_MONITORED>Y</NODE_MONITORED>
+  <SITENAME>GRNET</SITENAME>
+  <COUNTRY_NAME>Greece</COUNTRY_NAME>
+  <COUNTRY_CODE>GR</COUNTRY_CODE>
+  <ROC_NAME>EUDAT_REGISTRY</ROC_NAME>
+  <URL>https://epic.grnet.gr</URL>
+  <ENDPOINTS/>
+  <EXTENSIONS/>
+</SERVICE_ENDPOINT>
+
+<SERVICE_ENDPOINT PRIMARY_KEY="56GL">
+  <PRIMARY_KEY>56GL</PRIMARY_KEY> 
+  <GOCDB_PORTAL_URL>
+    https://creg.eudat.eu/view_portal/index.php?Page_Type=Service&id=5
+  </GOCDB_PORTAL_URL>
+  <BETA>N</BETA>
+  <SERVICE_TYPE>b2handle.handle.api</SERVICE_TYPE>
+  <HOST_IP>130.183.206.85</HOST_IP>
+  <CORE/>
+  <IN_PRODUCTION>Y</IN_PRODUCTION>
+  <NODE_MONITORED>Y</NODE_MONITORED>
+  <SITENAME>MPCDF</SITENAME>
+  <COUNTRY_NAME>Germany</COUNTRY_NAME>
+  <COUNTRY_CODE>DE</COUNTRY_CODE>
+  <ROC_NAME>EUDAT_REGISTRY</ROC_NAME>
+  <URL>https://epic.rzg.mpg.de/epic_v2/34155</URL>
+  <ENDPOINTS/>
+  <EXTENSIONS/>
+</SERVICE_ENDPOINT>
+```
+
```

Comment:
I am missing the information about the "get_service_group" call. THIS IS VERY IMPORTANT, because here we will show that two entries of the same Service Endpoint, can be referenced in different Service Groups